docs/docs/integrations/libraries/dbt/dbt-cloud-legacy.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Dagster & dbt Cloud (Legacy)
+sidebar_label: dbt Cloud integration (Legacy)
+description: Dagster allows you to run dbt Cloud jobs alongside other technologies. You can schedule them to run as a step in a larger pipeline and manage them as a data asset.
+tags: [dagster-supported, etl]
+source: https://github.com/dagster-io/dagster/tree/master/python_modules/libraries/dagster-dbt
+pypi: https://pypi.org/project/dagster-dbt/
+sidebar_custom_props:
+  logo: images/integrations/dbt/dbt.svg
+partnerlink:
+sidebar_position: 1000
+---
+
+import Deprecated from '@site/docs/partials/\_Deprecated.md';
```

Comment:
Can we mark as superseded instead? I'd need to add the superseded marker to the legacy integration too.